ansible/roles/debops.owncloud/tasks/ldap.yml

```diff
@@ -3,8 +3,6 @@
 
 - name: Check default ldap config exists
   command: php --file "{{ owncloud__app_home }}/occ" ldap:show-config ""
-  args:
-    executable: '/bin/sh'
```

Comment:
I'm not sure about the removal of the `executable` keywords... Isn't ownCloud user acount using `/usr/sbin/nologin` as a default shell? That would fail with recent Ansible which uses the user account shell by default, that's why these were even added.

Reply:
I see. Sounds good then. I need to update the Ansible used in GitLab CI tests and check all of them for any new deprecation warnings...